platform-sdk/docs/proposals/001-tss-rosters-design-proposal/001-tss-rosters-design-proposal.md

```diff
@@ -0,0 +1,265 @@
+# TSS - Roster Proposal
+
+---
+
+## Summary
+
+This proposal outlines the design and implementation of rosters within the Hedera Hashgraph system to support Threshold
+Signature Scheme (TSS) block signing. It details the roster's lifecycle, data storage, API, and necessary changes to the
+reconnect process.
+
+| Metadata           | Entities                                                  | 
+|--------------------|-----------------------------------------------------------|
+| Designers          | Kore, Anthony, Ed, Cody, Austin                           |
+| Functional Impacts | TSS, Cryptography, Reconnect                              |
+| Related Proposals  | TSS-Library, TSS-Roster, TSS-Ledger-Id, TSS-Block-Signing |
+| HIPS               | N/A                                                       |
+
+## Purpose and Context
+
+The introduction of the Threshold Signature Scheme (TSS) necessitates a new mechanism for managing node participation in
+consensus and block signing.
+Rosters, as immutable subsets of the address book, will provide this mechanism, ensuring efficient and secure key
+management for TSS operations.
+This proposal attempts to provide a specification for the behavior of rosters starting from their creation from the CAB
```

Comment:
I suggest to explicitly define project-related acronyms, such as the `CAB` here.